conans/test/functional/scm_test.py

```diff
@@ -149,6 +149,23 @@ def test_auto_subfolder(self):
         self.assertTrue(os.path.exists(os.path.join(folder, "mysub", "myfile.txt")))
         self.assertFalse(os.path.exists(os.path.join(folder, "mysub", "conanfile.py")))
 
+    def test_auto_conanfile_no_root(self):
+        """
+        Conanfile is not in the root of the repo: https://github.com/conan-io/conan/issues/3465
+        """
+        curdir = self.client.current_folder
+        conanfile = base.format(url="auto", revision="auto")
+        self.client.save({"conanfile.py": conanfile},
```

Comment:
You can run `self.client.save({"conan/conanfile.py": conanfile})

Reply:
And mix with "myfile.txt" save for clarity.

Reply:
much better, didn't thought about that 🙃